src/components/SnapshotSource.vue

```diff
@@ -58,13 +60,11 @@ export default {
     props: ['snapshotId', 'snapshotScrollPosition', 'viewportSize', 'highlight'],
     data: function() {
         return {
-            loaded: false
+            loaded: false,
+            snapshotUrl: `/api/snapshots/html/${this.snapshotId}`
```

Comment:
Not sure if I like that :-) I would rather use a computed property, but not some dynamical data property.

Reply:
done, reverted it back to the original code.